test/base/MockExportsDeclarationService.scala

```diff
@@ -26,13 +26,12 @@ import services.cache.{ExportsCacheService, ExportsDeclarationBuilder}
 
 import scala.concurrent.Future
 
-trait MockExportsCacheService extends MockitoSugar with ExportsDeclarationBuilder with BeforeAndAfterEach {
+trait MockExportsDeclarationService extends MockitoSugar with ExportsDeclarationBuilder with BeforeAndAfterEach {
```

Comment:
I'll come back to the previous name. Only model has different name, service is still `ExportsCacheService`

Reply:
Agree with Patryk